source/features/epochs/components/EpochInfo.scss

```diff
@@ -1,5 +1,6 @@
 .epochInfoContainer {
   margin: 61.5px 0;
+  font-family: ProximaNova;
```

Comment:
@daniloprates 
As you setup theme and styles, can you do that same for fonts also?
Currently we have such `font-family` definitions in many components individually, and this should be handled globally, and only specific fonts should be applied to specific individual components.

Reply:
@yakovkaravelov the way is setup we can simply copy the code from Zeplin and it'll work well. The font name is always `ProximaNova`, no quotes needed. The `font-weight` will define which file will be used. cc @DeeJayElly